Source/Libraries/GSF.Web/Security/Views/Login.cshtml

```diff
@@ -63,6 +64,11 @@
             loginPage = options.LoginPage;
         }
 
+        if (!string.IsNullOrWhiteSpace(options.LogoutPage))
```

Comment:
Not a big deal - but other code in this JavaScript file do not wrap to new line before `{`

Reply:
Good catch.